docs/tsp/typeServerProtocol.ts

```diff
@@ -115,43 +120,45 @@ export namespace TypeServerProtocol {
         // Name of the module the type comes from
```

Comment:
declaration's moduleName comment mentions this:
```
// The dot-separated import name for the file that
// contains the declaration (may not be definitive
// because a source file can be accessed via different
// import names in some cases).
```

I imagine moduleName means the same thing here? If it, should both of these be a `ModuleName` type which is a list of `nameParts`?

This makes assumptions about the project root. TBD if problems arise from this as I'm looking at the rest of this.

Reply:
Good idea. I'll incorporate that next time. Make this a name parts list.